### ISSUE TITLE
Inconsistent vocabulary versions

### ISSUE THREAD

**Issue by cukarthik** [OHDSI/Vocabulary-v5.0#406]
I downloaded the vocabulary in twice in Sept (v5.0 17-SEP-20 and v5.0 25-SEP-20) and Oct (v5.0 02-OCT-20). In the v5.0 25-SEP-20, I see the new 'Type Concept' vocabulary in the vocabulary file; however it does not exist in the v5.0 17-SEP-20 and v5.0 02-OCT-20 versions. Athena shows the correct [vocabulary](https://athena.ohdsi.org/search-terms/terms?vocabulary=Type+Concept&standardConcept=Standard). Was there something that changed in the vocabulary script?

**Comment by mik-ohdsi** [comment]
Hi Karthik (@cukarthik) , we cannot easily reproduce this issue. The type concepts were added in a late August [release](https://github.com/OHDSI/Vocabulary-v5.0/releases/tag/v20200821_1598012496).
At least per the release notes, I wouldn't see that we changed something about them since. If it was a problem, it seems to be temporary but I will let the team know so we can double check if there is a general issue. 
Thanks ~ Mik
